docs/usage/configuration-options.md

```diff
@@ -1291,29 +1291,29 @@ A similar one could strip leading `v` prefixes:
 }
 ```
 
-## fetchReleaseNotes
+## fetchChangeLogs
 
-Use this config option to configure release notes fetching.
+Use this config option to configure changelogs/release notes fetching.
 The available options are:
 
-- `off` - disable release notes fetching
-- `branch` - fetch release notes while creating/updating branch
-- `pr`(default) - fetches release notes while creating/updating pull-request
+- `off` - disable changelogs fetching
+- `branch` - fetch changelogs while creating/updating branch
+- `pr`(default) - fetches changelogs while creating/updating pull-request
 
-It is not recommended to set fetchReleaseNotes=branch unless you are embedding release notes in commit information, because it results in a performance decrease.
+It is not recommended to set fetchChangeLogs=branch unless you are embedding changelogs in commit information, because it results in a performance decrease.
```

Comment:
```suggestion
Avoid setting `fetchChangeLogs=branch`, because this slows down Renovate.
But if you're embedding changelogs in commit information, you may use `fetchChangeLogs=branch`.
```